tests/integration/helpers.py

```diff
@@ -174,7 +206,6 @@ def project_exists(self, project_name):
             pipeline_node,
             (repository_iri, SwarmUI.pipelines, pipeline_node),
         ])
-        await self.scheduler_complete(pipeline_id)
```

Comment:
If you remove it here, you need to add it to every test that uses `create_pipeline`

Reply:
I forgot this one, I don't know why I removed it, I will put it back.